docs/core/tools/dotnet-build.md

```diff
@@ -42,15 +50,15 @@ Whether the project is executable or not is determined by the `<OutputType>` pro
 </PropertyGroup>
 ```
 
-To produce a library, omit the `<OutputType>` property. The main difference in built output is that the IL DLL for a library doesn't contain entry points and can't be executed.
+To produce a library, omit the `<OutputType>` property or change its value to `Library`. The IL DLL for a library doesn't contain entry points and can't be executed.
 
 ### MSBuild
 
 `dotnet build` uses MSBuild to build the project, so it supports both parallel and incremental builds. For more information, see [Incremental Builds](/visualstudio/msbuild/incremental-builds).
 
 In addition to its options, the `dotnet build` command accepts MSBuild options, such as `-p` for setting properties or `-l` to define a logger. For more information about these options, see the [MSBuild Command-Line Reference](/visualstudio/msbuild/msbuild-command-line-reference). Or you can also use the [dotnet msbuild](dotnet-msbuild.md) command.
 
-Running `dotnet build` is equivalent to `dotnet msbuild -restore -target:Build`.
+Running `dotnet build` is equivalent to running `dotnet msbuild -restore`; however, the default verbosity of the output is different.
```

Comment:
@dsplaisted how do you feel about this comment made here?
https://github.com/dotnet/docs/pull/15196#discussion_r335136282

Is there something we should note here?

Reply:
I don't think so.  We are showing the safe way to do it `-restore` here.